front/src/common/api/generatedEditoastApi.ts

```diff
@@ -826,6 +864,37 @@ const injectedRtkApi = api
     overrideExisting: false,
   });
 export { injectedRtkApi as generatedEditoastApi };
+export type GetAuthzListRolesApiResponse =
+  /** status 200 List of supported application roles */ string[];
+export type GetAuthzListRolesApiArg = void;
+export type GetAuthzRolesMeApiResponse =
+  /** status 200 List the roles of the issuer of the request */ {
+    builtin: BuiltinRole[];
+  };
+export type GetAuthzRolesMeApiArg = void;
+export type GetAuthzRolesByUserIdApiResponse = /** status 200 List the roles of a user */ {
+  builtin: BuiltinRole[];
+};
+export type GetAuthzRolesByUserIdApiArg = {
+  /** A user ID (not to be mistaken for its identity, cf. editoast user model documentation) */
+  userId: number;
+};
+export type PostAuthzRolesByUserIdApiResponse = unknown;
```

Comment:
Don't we want a success message for when we will have an admin interface ?

Reply:
I'd say `204 NO CONTENT` is enough of a response in this case since the sole purpose of this endpoint is to grant a bunch of roles. In any case this whole API is likely to change when we'll implement the admin panel. But in the meantime, I figured the status code would be enough especially since the frontend will never call the endpoint (roles will be granted manually by the ops/support). Wdyt?

Reply:
Totally fine yes :)